packages/manager/modules/nasha/src/components/metrics/metrics.template.html

```diff
@@ -0,0 +1,103 @@
+<div class="nasha-components-metrics col-xs-12 well top-space-p20">
+    <div class="row">
+        <div class="col-xs-12 col-lg-7">
+            <dl class="dl-horizontal">
```

Comment:
**suggestion**: `.oui-description` can be used to maintain the similar User experience. With this usage, `.bottom-space-m4` can also be removed.

Reply:
I have to admit I didn't touch that part of the code, but I would stick to the original class. See for yourself :
Upper: with `dl-horizontal`
Lower: with `ui-description oui-description_horizontal`
![image](https://user-images.githubusercontent.com/3984480/162893464-9fa0aad8-8075-4a4a-bfb0-a24055056ce4.png)
Also, `dl-horizontal` seems to be widely used throughout the code base